### PR TITLE
Tracepoints for mempool in tapyrus

### PR DESCRIPTION
**Pull request by Naviabheeman**
Port mempool add, remove, replace and rejected UDST tracepoints from bitcoin

### CODE REVIEW

File: contrib/tracing/mempool_monitor.py [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+// strprintf("mandatory-script-verify-flag-failed (%s)", ScriptErrorString(check.GetScriptError()))
+#define MAX_REJECT_REASON_LENGTH        118
+// The longest string returned by RemovalReasonToString() is 'sizelimit'
+#define MAX_REMOVAL_REASON_LENGTH       9
+#define HASH_LENGTH                     32
+
+struct added_event
+{
+  u8    hash[HASH_LENGTH];
+  u64   vsize;
```
Comment (azuchi) [inline contrib/tracing/mempool_monitor.py:30]: It would be better to also port the following PR:
https://github.com/bitcoin/bitcoin/pull/23962

**Review by azuchi — CHANGES_REQUESTED** [review]
Similar to Bitcoin, the following documentation needs to be updated:

* contrib/tracing/README.md
* doc/tracing.md

File: contrib/tracing/mempool_monitor.py (hunk 1 above)
Comment (Naviabheeman) [reply contrib/tracing/mempool_monitor.py]: Done!
We removed tx weight related code from tapyrus. So only the size calculation within mempool needed to be updated.

**Comment by Naviabheeman** [comment]
> Similar to Bitcoin, the following documentation needs to be updated:
> 
> * contrib/tracing/README.md
> * doc/tracing.md

done!

**Review by azuchi — APPROVED** [review]

**Pull request merged**